### DISCUSSION TITLE
[АС] Бой на кораблях ЦЛК 25 обычка

### DISCUSSION THREAD

**Discussion by xBarselonx** [trickerer/Trinity-Bots#935] [General]
При первом пуле когда противоположная фракция вступает в бой у меня вайпантся весь рейд с ботами и я в том числе. С чем это может быть связано?

**Comment by trickerer** [comment]
На Тринити не знаю, а на Азероткоре есть какая-то проверка на возможность начала битвы - какое-то количество игроков должно быть на палубе (не на пушках), иначе - вайп сразу. Почему второй пул возможен - не могу знать. Говорю так, поскольку код не смотрел, помню только что в истории коммитов что-то такое было, добавлено месяца 3 назад

**Comment by xBarselonx** [comment]
Раньше такого не было. Я садился на пушку набирал энергию на пушке, палил по короблю и все норм. Не помню когда началась эта проблема. Сейчас пробовал перед пулом садиться на пушки и просто так, все равно заканчивалось вайпом. Что самое интересное со второго пула все норм.
